### PR TITLE
Fix accountBalance format and file encoding

### PR DESCRIPTION
**Pull request by mincong-h**
## Account Balance

**TL;DR: format(amount)=FR, format(accountbalance)=ISO** 

Previously, the format of the column "accountbalance" was written in French format:

* "1 000,00"
* 370,00

Now it is using the ISO format without thousands separator:

* 1000.00
* 370.00

but the other column "amount" remains unchanged -- i.e. still in French format. To fix this, we consider the column "accountbalance" as string when reading the CSV file and cast it ourselves.

## Encoding

Previously the encoding of the file is ISO-8859-1, it had been changed to UTF-8 now. When using ISO-8859-1, we can see the problem from this error, where `dateOp` was considered as `ï»¿dateOp`:

```
finance_toolkit.pipeline.PipelineDataError: Failed to read new Boursorama data. Details:
  path=/data/source/export-operations-11-06-2022_09-52-55.csv
  headers=ï»¿dateOp;dateVal;label;category;categoryParent;amount;comment;accountNum;accountLabel;accountbalance
  pandas_kwargs={'decimal': ',', 'delimiter': ';', 'dtype': {'accountNum': 'str'}, 'encoding': 'ISO-8859-1', 'parse_dates': ['dateOp', 'dateVal'], 'skipinitialspace': True, 'thousands': ' '}
  pandas_error=Missing column provided to 'parse_dates': 'dateOp'
```

### CODE REVIEW

**Comment by mincong-h** [comment]
It's related to https://github.com/mincong-h/finance-toolkit/issues/40
